packages/App/src/store/saga/debitsSaga.tsx

```diff
@@ -87,6 +88,9 @@ export function* addNewDebit(
   } catch (error) {
     console.log('newDebits', error);
     yield put(addNewDebitFailed((error as Error).message));
+    if ((error as ErrorFetch).code === 401) {
+      yield put(logOutActionSuccess());
```

Comment:
Is this an action with type `LOGOUT`, that is handled in `packages/App/src/store/saga/userLogIn.tsx`? If not, it should be changed

Reply:
In every place you want to logout you should use one action and handle it accordingly in `packages/App/src/store/saga/userLogIn.tsx`